src/components/SeniorView.tsx

```diff
@@ -0,0 +1,41 @@
+"use client";
+
+import { Senior, User } from "@prisma/client";
+import SearchableContainer from "./SearchableContainer";
+import { UserTile } from "./TileGrid";
+import AddSenior from "./AddSenior";
+import { useState } from "react";
+
+type SeniorViewProps = {
+  seniors: Senior[];
+  students: User[];
+};
+
+export const SeniorView = ({ seniors, students }: SeniorViewProps) => {
+  const [seniorsState, setSeniorsState] = useState(seniors);
+  const [showAddSeniorPopUp, setShowAddSeniorPopUp] = useState(false);
+  const [seniorPatch, setSeniorPatch] = useState("");
+
+  return (
+    <SearchableContainer<Senior>
+      addElementComponent={
+        <AddSenior
```

Comment:
We need to add a `key` property here, because `AddSenior` is added to the list along with the elements.